<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ExportDialog</class>
    <widget class="QDialog" name="ExportDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>614</width>
                <height>358</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
            </property>
            <item row="9" column="0" colspan="2">
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
                    </property>
                </widget>
            </item>
            <item row="1" column="0" colspan="2">
                <widget class="QWidget" name="pathGroup" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QLabel" name="label_13">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>File:</string>
                                </property>
                                <property name="buddy">
                                    <cstring>textFile</cstring>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="textFile"/>
                        </item>
                        <item>
                            <widget class="QToolButton" name="fileBrowse">
                                <property name="text">
                                    <string>...</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item row="3" column="0">
                <widget class="QTabWidget" name="tabWidget">
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="QWidget" name="tab1">
                        <attribute name="title">
                            <string>Settings</string>
                        </attribute>
                        <layout class="QGridLayout" name="gridLayout_2">
                            <item row="0" column="0">
                                <widget class="QGroupBox" name="modeFormat">
                                    <property name="title">
                                        <string>Format</string>
                                    </property>
                                    <layout class="QFormLayout" name="formLayout_2">
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item row="0" column="0">
                                            <widget class="QRadioButton" name="rbText">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="text">
                                                    <string>Text (only filenames)</string>
                                                </property>
                                                <property name="checked">
                                                    <bool>true</bool>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="1" column="0">
                                            <widget class="QRadioButton" name="rbCSV">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="text">
                                                    <string>CSV (with size and datetime)</string>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QGroupBox" name="groupBox_2">
                                    <property name="title">
                                        <string>Settings</string>
                                    </property>
                                    <layout class="QGridLayout" name="gridLayout_4">
                                        <item row="6" column="0">
                                            <widget class="QLabel" name="label_12">
                                                <property name="text">
                                                    <string>Extra arguments:</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="2" column="0">
                                            <widget class="QLabel" name="label_18">
                                                <property name="text">
                                                    <string>Maximum age (s or ms|s|m|h|d|w|M|y suffix)</string>
                                                </property>
                                                <property name="buddy">
                                                    <cstring>textMaxAge</cstring>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="3" column="1">
                                            <widget class="QSpinBox" name="spinMaxDepth">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="alignment">
                                                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                                </property>
                                                <property name="maximum">
                                                    <number>999999</number>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="1" column="0">
                                            <widget class="QLabel" name="label_16">
                                                <property name="text">
                                                    <string>Minimum age (s or ms|s|m|h|d|w|M|y suffix)</string>
                                                </property>
                                                <property name="buddy">
                                                    <cstring>textMinAge</cstring>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="0" column="1">
                                            <widget class="QLineEdit" name="textMinSize">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="1" column="1">
                                            <widget class="QLineEdit" name="textMinAge">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="0" column="0">
                                            <widget class="QLabel" name="label_15">
                                                <property name="text">
                                                    <string>Minimum size (KiB or b|k|M|G suffix)</string>
                                                </property>
                                                <property name="buddy">
                                                    <cstring>textMinSize</cstring>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="2" column="1">
                                            <widget class="QLineEdit" name="textMaxAge">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="3" column="0">
                                            <widget class="QLabel" name="label_17">
                                                <property name="text">
                                                    <string>Maximum depth</string>
                                                </property>
                                                <property name="buddy">
                                                    <cstring>spinMaxDepth</cstring>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="7" column="0" colspan="2">
                                            <widget class="QLineEdit" name="textExtra"/>
                                        </item>
                                        <item row="5" column="0" colspan="2">
                                            <widget class="QCheckBox" name="checkSameFilesystem">
                                                <property name="text">
                                                    <string>Don't cross filesystem boundaries</string>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="tab4">
                        <attribute name="title">
                            <string>Exclude</string>
                        </attribute>
                        <layout class="QVBoxLayout" name="verticalLayout_3">
                            <item>
                                <widget class="QPlainTextEdit" name="textExclude">
                                    <property name="lineWrapMode">
                                        <enum>QPlainTextEdit::NoWrap</enum>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <tabstops>
        <tabstop>textFile</tabstop>
        <tabstop>fileBrowse</tabstop>
        <tabstop>tabWidget</tabstop>
        <tabstop>rbText</tabstop>
        <tabstop>rbCSV</tabstop>
        <tabstop>textMinSize</tabstop>
        <tabstop>textMinAge</tabstop>
        <tabstop>textMaxAge</tabstop>
        <tabstop>spinMaxDepth</tabstop>
        <tabstop>checkSameFilesystem</tabstop>
        <tabstop>textExtra</tabstop>
        <tabstop>textExclude</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
